pkg/logs/message/message.go

```diff
@@ -91,6 +91,7 @@ type MessageContent struct { //nolint:revive
 	// structured content
 	structuredContent StructuredContent
 	State             MessageContentState
+	Tags              []string
```

Comment:
Spoke offline - but we should move this to `ParsingExtra`

Reply:
Moved!